src/services/tfjs-mobilenet-service.ts

```diff
@@ -73,8 +73,9 @@ interface MobilenetEmbedding extends MobilenetParams {
  * @return a reference number to the model, maintained by the `ResourceManager`.
  */
 const load = async ({version = 1, alpha = 1.0}: MobilenetParams): Promise<Reference> => {
-  log('Loading tfjs...');
-  const tf = await requireTf();
+  log('Loading tensorflow');
+  await requireTf();
+
   log('Loading MobileNet...');
   await dynamicScript(modelUrl);
```

Comment:
Noting: as discussed, we should access mobileNet via some encapsulation (i.e. requireMobileNet), but we are ignoring this for now as @alxrsngrtn plans to use mobileNet via TF instead of as it's own executable.